server/boot/challenge.js

```diff
@@ -36,7 +38,11 @@ function buildUserUpdate(
         completedChallenge: challengeId
       }
     };
-    finalChallenge = completedChallenge;
+    finalChallenge = {
+      ...completedChallenge,
+      lastUpdated: completedChallenge.completedDate,
+      submitCount: 1
```

Comment:
This value can come from the same var as above

Reply:
Looks so